simpleclient/src/main/java/io/prometheus/client/Info.java

```diff
@@ -1,30 +1,24 @@
 package io.prometheus.client;
 
-import io.prometheus.client.CKMSQuantiles.Quantile;
-
-import java.io.Closeable;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.TreeMap;
-import java.util.concurrent.Callable;
-import java.util.concurrent.TimeUnit;
 
 /**
  * Info metric, key-value pairs.
  *
- * Examples of Info include, build information, version information, and potential target metadata,
- * The first provided state will be the default.
+ * Examples of Info include build information, version information, and potential target metadata.
+ * The first provided state will be the default. The string "_info" will be appended to the name.
```

Comment:
This is misleading, the metric name is not changed. The sample name is what will get the suffix.

Reply:
Sorry, I don't have a complete grasp of all the OM concepts yet. Should it say `... appended to the sample name.` instead?

Reply:
The term "metric" has a few different meanings in Prometheus, OM tries to fix that by defining it. That should do the trick.

Reply:
Changed!

Also removed `The first provided state will be the default.` as I believe that was a copy-paste error from Enumeration.